controllers/users.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectAllArticles } = require("../models/users.model.js");
+
+
+exports.getAllUsers = (req, res, next) => {
+    selectAllArticles().then((users) => {
+        res.status(200).send(users);
```

Comment:
send data on a key of users, i.e. `.send({ users });`
_note: this may require changes in the tests for accessing the data_

Reply:
Thank you, missed that one